### PR TITLE
Database swap prep

### PR DESCRIPTION
**Pull request by geophray**
@asmit222 would  you please review? 

This PR includes my data generation script and some restructuring of my db directory in preparation for swapping out databases.

### CODE REVIEW

**Review by asmit222 — APPROVED** [review]
looks good

**Comment by geophray** [comment]
Awesome... would you mind approving it? I can't merge until I get your official seal of approval. :)

**Comment by geophray** [comment]
Oh wait... I think @stephenJohnGiardina just needs to update our org settings. Disregard my last comment. Thanks Austin!

**Pull request merged**
